frontend/pages/_app.tsx

```diff
@@ -46,8 +46,18 @@ export default function App({
   const router = useRouter();
   const { locale, defaultLocale, pathname } = router;
   const messages = locales[locale];
+  const theme = extendTheme({
+    colors: {
+      brand: {
+        100: "#2d9cdc", // main blue
+        200: "#96cfee", // light blue
+        300: "gray.600", // dark grey text
+        400: "#4e93bb", // dark blue shadow
+      },
+    },
+  });
   return (
-    <ChakraProvider>
+    <ChakraProvider theme={theme}>
       <IntlProvider
         locale={locale}
         defaultLocale={defaultLocale}
```

Comment:
Great work with this! Should be super useful when we use it for other components, one suggestion I would make is to keep theme in it's own file, perhaps `components/Theme`? and import it into this file and `.storybook/preview.tsx`, would help keep each file independent of each other!